cmd/shortener/config_test.go

```diff
@@ -52,3 +54,44 @@ func TestParse(t *testing.T) {
 		})
 	}
 }
+
+func TestNewConfigs(t *testing.T) {
+	cfg := NewConfigs()
+	if cfg == nil {
+		t.Errorf("Ожидали %v, пришли %v", nil, cfg)
+	}
+
+}
+
+func TestLoadConfig_Successful(t *testing.T) {
+	// Создаём временный JSON-файл для теста
+	configData := Configs{
+		AddrServer: ":9000",
+		BaseURL:    "http://example.com",
+		LogLevel:   "debug",
+		PathFile:   "/tmp/storage",
+		AddrDB:     "postgres://user:pass@localhost:5432/db",
+		HTTPS:      new(bool),
+	}
+
+	*configData.HTTPS = true // устанавливаем значение для HTTPS в true
+
+	file, err := os.CreateTemp("", "config_test.json")
+	if err != nil {
+		t.Fatalf("Не удалось создать временный файл: %v", err)
+	}
+	defer os.Remove(file.Name())
+
+	// Сохраняем конфигурационные данные в JSON-файл
+	if err := json.NewEncoder(file).Encode(configData); err != nil {
+		t.Fatalf("Не удалось записать данные в файл: %v", err)
+	}
+
+	// Инициализируем тестируемую структуру Configs и устанавливаем путь к файлу конфигурации
+	cfg := NewConfigs()
+	cfg.ConfigFile = file.Name()
+	err = cfg.loadConfig()
+	if err != nil {
+		t.Errorf("Ожидали %v, пришли %v", nil, err)
+	}
```

Comment:
Мне кажется проще сделать так и не будет двусмысленности. Вот здесь есть неплохие комментарии от команды Go https://go.dev/wiki/TestComments#table-driven-tests-vs-multiple-test-functions.
```suggestion
	if err != nil {
		t.Errorf("loadConfig() = %v", err)
	}
```

Reply:
спасибо за рекомендацию, применил